exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -229,9 +229,15 @@ def __init__(
 
         self._headers = headers or environ.get(OTEL_EXPORTER_OTLP_HEADERS)
         if isinstance(self._headers, str):
-            self._headers = tuple(
-                tuple(item.split("=")) for item in self._headers.split(",")
-            )
+            temp_headers = []
+            for header_pair in self._headers.split(","):
+                key, value = header_pair.split("=", maxsplit=1)
+                key = key.strip().lower()
+                value = value.strip()
```

Comment:
I'm sure if we should strip individual keys and values. Generally we just strip a pair as people tend to add whitespace after comma but don't do it before/after `=`. That said, I can't think of any case where someone would want whitespace at the beginning or end of a key/value so I guess it's fine.